api/lib/domain/services/mail-service.js

```diff
@@ -12,97 +12,97 @@ import { config } from '../../config.js';
-  let pixName;
-  let accountCreationEmailSubject;
-  let variables;
+  const mailerConfig = _getMailerConfig(locale);
 
-  if (locale === FRENCH_SPOKEN) {
-    variables = {
-      homeName: PIX_HOME_NAME_INTERNATIONAL,
-      homeUrl: PIX_HOME_URL_INTERNATIONAL_FRENCH_SPOKEN,
-      redirectionUrl: redirectionUrl || `${config.domain.pixApp + config.domain.tldOrg}/connexion/?lang=fr`,
-      helpdeskUrl: HELPDESK_FRENCH_SPOKEN,
-      displayNationalLogo: false,
-      ...frTranslations['pix-account-creation-email'].params,
-    };
-
-    pixName = frTranslations['email-sender-name']['pix-app'];
-    accountCreationEmailSubject = frTranslations['pix-account-creation-email'].subject;
-  } else if (locale === ENGLISH_SPOKEN) {
-    variables = {
-      homeName: PIX_HOME_NAME_INTERNATIONAL,
-      homeUrl: PIX_HOME_URL_INTERNATIONAL_ENGLISH_SPOKEN,
-      redirectionUrl: redirectionUrl || `${config.domain.pixApp + config.domain.tldOrg}/connexion/?lang=en`,
-      helpdeskUrl: HELPDESK_ENGLISH_SPOKEN,
-      displayNationalLogo: false,
-      ...enTranslations['pix-account-creation-email'].params,
-    };
-
-    pixName = enTranslations['email-sender-name']['pix-app'];
-    accountCreationEmailSubject = enTranslations['pix-account-creation-email'].subject;
-  } else if (locale === DUTCH_SPOKEN) {
-    variables = {
-      homeName: PIX_HOME_NAME_INTERNATIONAL,
-      homeUrl: PIX_HOME_URL_INTERNATIONAL_DUTCH_SPOKEN,
-      redirectionUrl: redirectionUrl || `${config.domain.pixApp + config.domain.tldOrg}/connexion/?lang=nl`,
-      helpdeskUrl: HELPDESK_DUTCH_SPOKEN,
-      displayNationalLogo: false,
-      ...nlTranslations['pix-account-creation-email'].params,
-    };
-
-    pixName = nlTranslations['email-sender-name']['pix-app'];
-    accountCreationEmailSubject = nlTranslations['pix-account-creation-email'].subject;
-  } else if (locale === SPANISH_SPOKEN) {
-    variables = {
-      homeName: PIX_HOME_NAME_INTERNATIONAL,
-      homeUrl: PIX_HOME_URL_INTERNATIONAL_ENGLISH_SPOKEN,
-      redirectionUrl: redirectionUrl || `${config.domain.pixApp + config.domain.tldOrg}/connexion/?lang=es`,
-      helpdeskUrl: HELPDESK_ENGLISH_SPOKEN,
-      displayNationalLogo: false,
-      ...esTranslations['pix-account-creation-email'].params,
-    };
-    pixName = esTranslations['email-sender-name']['pix-app'];
-    accountCreationEmailSubject = esTranslations['pix-account-creation-email'].subject;
-  } else {
-    variables = {
-      homeName: PIX_HOME_NAME_FRENCH_FRANCE,
-      homeUrl: PIX_HOME_URL_FRENCH_FRANCE,
-      redirectionUrl: redirectionUrl || `${config.domain.pixApp + config.domain.tldFr}/connexion`,
-      helpdeskUrl: HELPDESK_FRENCH_FRANCE,
-      displayNationalLogo: true,
-      ...frTranslations['pix-account-creation-email'].params,
-    };
-
-    pixName = frTranslations['email-sender-name']['pix-app'];
-    accountCreationEmailSubject = frTranslations['pix-account-creation-email'].subject;
-  }
+  const templateParams = {
```

Comment:
Chipotage : tu décides si tu veux prendre en compte ou non 😉 

```suggestion
  const templateVariables = {
```